diff -ruN a/Blocks/Tables.mo b/Blocks/Tables.mo
--- a/Blocks/Tables.mo
+++ b/Blocks/Tables.mo
@@ -1035,7 +1035,7 @@
       input Real pre_nextTimeEvent "Pre-value of (scaled) next time event in table";
       output Real y "Interpolated value";
       external "C" y = ModelicaStandardTables_CombiTimeTable_getValue(tableID, icol, timeIn, nextTimeEvent, pre_nextTimeEvent)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
       annotation (derivative(
           noDerivative=nextTimeEvent,
           noDerivative=pre_nextTimeEvent) = getDerTimeTableValue);
@@ -1051,7 +1051,7 @@
       input Real pre_nextTimeEvent "Pre-value of (scaled) next time event in table";
       output Real y "Interpolated value";
       external "C" y = ModelicaStandardTables_CombiTimeTable_getValue(tableID, icol, timeIn, nextTimeEvent, pre_nextTimeEvent)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getTimeTableValueNoDer;
 
     pure function getTimeTableValueNoDer2
@@ -1064,7 +1064,7 @@
       input Real pre_nextTimeEvent "Pre-value of (scaled) next time event in table";
       output Real y "Interpolated value";
       external "C" y = ModelicaStandardTables_CombiTimeTable_getValue(tableID, icol, timeIn, nextTimeEvent, pre_nextTimeEvent)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
       annotation (derivative(
           noDerivative=nextTimeEvent,
           noDerivative=pre_nextTimeEvent) = getDerTimeTableValueNoDer);
@@ -1081,7 +1081,7 @@
       input Real der_timeIn "Derivative of (scaled) time value";
       output Real der_y "Derivative of interpolated value";
       external "C" der_y = ModelicaStandardTables_CombiTimeTable_getDerValue(tableID, icol, timeIn, nextTimeEvent, pre_nextTimeEvent, der_timeIn)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
       annotation (derivative(
           order=2,
           noDerivative=nextTimeEvent,
@@ -1099,7 +1099,7 @@
       input Real der_timeIn "Derivative of (scaled) time value";
       output Real der_y "Derivative of interpolated value";
       external "C" der_y = ModelicaStandardTables_CombiTimeTable_getDerValue(tableID, icol, timeIn, nextTimeEvent, pre_nextTimeEvent, der_timeIn)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getDerTimeTableValueNoDer;
 
     pure function getDer2TimeTableValue
@@ -1114,7 +1114,7 @@
       input Real der2_timeIn "Second derivative of (scaled) time value";
       output Real der2_y "Second derivative of interpolated value";
       external "C" der2_y = ModelicaStandardTables_CombiTimeTable_getDer2Value(tableID, icol, timeIn, nextTimeEvent, pre_nextTimeEvent, der_timeIn, der2_timeIn)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getDer2TimeTableValue;
 
     pure function getTimeTableTmin
@@ -1123,7 +1123,7 @@
       input Modelica.Blocks.Types.ExternalCombiTimeTable tableID "External table object";
       output Real timeMin "Minimum abscissa value in table";
       external "C" timeMin = ModelicaStandardTables_CombiTimeTable_minimumTime(tableID)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getTimeTableTmin;
 
     pure function getTimeTableTmax
@@ -1132,7 +1132,7 @@
       input Modelica.Blocks.Types.ExternalCombiTimeTable tableID "External table object";
       output Real timeMax "Maximum abscissa value in table";
       external "C" timeMax = ModelicaStandardTables_CombiTimeTable_maximumTime(tableID)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getTimeTableTmax;
 
     pure function getNextTimeEvent
@@ -1142,7 +1142,7 @@
       input Real timeIn "(Scaled) time value";
       output Real nextTimeEvent "(Scaled) next time event in table";
       external "C" nextTimeEvent = ModelicaStandardTables_CombiTimeTable_nextTimeEvent(tableID, timeIn)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getNextTimeEvent;
 
     pure function getTable1DValue "Interpolate 1-dim. table defined by matrix"
@@ -1152,7 +1152,7 @@
       input Real u "Abscissa value";
       output Real y "Interpolated value";
       external "C" y = ModelicaStandardTables_CombiTable1D_getValue(tableID, icol, u)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
       annotation (derivative = getDerTable1DValue);
     end getTable1DValue;
 
@@ -1164,7 +1164,7 @@
       input Real u "Abscissa value";
       output Real y "Interpolated value";
       external "C" y = ModelicaStandardTables_CombiTable1D_getValue(tableID, icol, u)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getTable1DValueNoDer;
 
     pure function getTable1DValueNoDer2
@@ -1175,7 +1175,7 @@
       input Real u "Abscissa value";
       output Real y "Interpolated value";
       external "C" y = ModelicaStandardTables_CombiTable1D_getValue(tableID, icol, u)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
       annotation (derivative = getDerTable1DValueNoDer);
     end getTable1DValueNoDer2;
 
@@ -1188,7 +1188,7 @@
       input Real der_u "Derivative of abscissa value";
       output Real der_y "Derivative of interpolated value";
       external "C" der_y = ModelicaStandardTables_CombiTable1D_getDerValue(tableID, icol, u, der_u)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
       annotation (derivative(order=2) = getDer2Table1DValue);
     end getDerTable1DValue;
 
@@ -1201,7 +1201,7 @@
       input Real der_u "Derivative of abscissa value";
       output Real der_y "Derivative of interpolated value";
       external "C" der_y = ModelicaStandardTables_CombiTable1D_getDerValue(tableID, icol, u, der_u)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getDerTable1DValueNoDer;
 
     pure function getDer2Table1DValue
@@ -1214,7 +1214,7 @@
       input Real der2_u " Second derivative of abscissa value";
       output Real der2_y "Second derivative of interpolated value";
       external "C" der2_y = ModelicaStandardTables_CombiTable1D_getDer2Value(tableID, icol, u, der_u, der2_u)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getDer2Table1DValue;
 
     pure function getTable1DAbscissaUmin
@@ -1223,7 +1223,7 @@
       input Modelica.Blocks.Types.ExternalCombiTable1D tableID "External table object";
       output Real uMin "Minimum abscissa value in table";
       external "C" uMin = ModelicaStandardTables_CombiTable1D_minimumAbscissa(tableID)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getTable1DAbscissaUmin;
 
     pure function getTable1DAbscissaUmax
@@ -1232,7 +1232,7 @@
       input Modelica.Blocks.Types.ExternalCombiTable1D tableID "External table object";
       output Real uMax "Maximum abscissa value in table";
       external "C" uMax = ModelicaStandardTables_CombiTable1D_maximumAbscissa(tableID)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getTable1DAbscissaUmax;
 
     pure function getTable2DValue "Interpolate 2-dim. table defined by matrix"
@@ -1242,7 +1242,7 @@
       input Real u2 "Value of second independent variable";
       output Real y "Interpolated value";
       external "C" y = ModelicaStandardTables_CombiTable2D_getValue(tableID, u1, u2)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
       annotation (derivative = getDerTable2DValue);
     end getTable2DValue;
 
@@ -1254,7 +1254,7 @@
       input Real u2 "Value of second independent variable";
       output Real y "Interpolated value";
       external "C" y = ModelicaStandardTables_CombiTable2D_getValue(tableID, u1, u2)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getTable2DValueNoDer;
 
     pure function getTable2DValueNoDer2
@@ -1265,7 +1265,7 @@
       input Real u2 "Value of second independent variable";
       output Real y "Interpolated value";
       external "C" y = ModelicaStandardTables_CombiTable2D_getValue(tableID, u1, u2)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
       annotation (derivative = getDerTable2DValueNoDer);
     end getTable2DValueNoDer2;
 
@@ -1279,7 +1279,7 @@
       input Real der_u2 "Derivative of second independent variable";
       output Real der_y "Derivative of interpolated value";
       external "C" der_y = ModelicaStandardTables_CombiTable2D_getDerValue(tableID, u1, u2, der_u1, der_u2)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
       annotation (derivative(order=2) = getDer2Table2DValue);
     end getDerTable2DValue;
 
@@ -1293,7 +1293,7 @@
       input Real der_u2 "Derivative of second independent variable";
       output Real der_y "Derivative of interpolated value";
       external "C" der_y = ModelicaStandardTables_CombiTable2D_getDerValue(tableID, u1, u2, der_u1, der_u2)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getDerTable2DValueNoDer;
 
     pure function getDer2Table2DValue
@@ -1308,7 +1308,7 @@
       input Real der2_u2 "Second derivative of second independent variable";
       output Real der2_y "Second derivative of interpolated value";
       external "C" der2_y = ModelicaStandardTables_CombiTable2D_getDer2Value(tableID, u1, u2, der_u1, der_u2, der2_u1, der2_u2)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getDer2Table2DValue;
 
     pure function getTable2DAbscissaUmin
@@ -1317,7 +1317,7 @@
       input Modelica.Blocks.Types.ExternalCombiTable2D tableID "External table object";
       output Real uMin[2] "Minimum abscissa value in table";
       external "C" ModelicaStandardTables_CombiTable2D_minimumAbscissa(tableID, uMin)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getTable2DAbscissaUmin;
 
     pure function getTable2DAbscissaUmax
@@ -1326,7 +1326,7 @@
       input Modelica.Blocks.Types.ExternalCombiTable2D tableID "External table object";
       output Real uMax[2] "Maximum abscissa value in table";
       external "C" ModelicaStandardTables_CombiTable2D_maximumAbscissa(tableID, uMax)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end getTable2DAbscissaUmax;
   end Internal;
   annotation (Documentation(info="<html>
diff -ruN a/Blocks/Types.mo b/Blocks/Types.mo
--- a/Blocks/Types.mo
+++ b/Blocks/Types.mo
@@ -132,14 +132,14 @@
             timeEvents,
             verboseRead,
             delimiter,
-            nHeaderLines) annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+            nHeaderLines) annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end constructor;
 
     function destructor "Terminate 1-dim. table where first column is time"
       extends Modelica.Icons.Function;
       input ExternalCombiTimeTable externalCombiTimeTable;
     external "C" ModelicaStandardTables_CombiTimeTable_close(
-        externalCombiTimeTable) annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        externalCombiTimeTable) annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end destructor;
 
   end ExternalCombiTimeTable;
@@ -172,14 +172,14 @@
             extrapolation,
             verboseRead,
             delimiter,
-            nHeaderLines) annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+            nHeaderLines) annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end constructor;
 
     function destructor "Terminate 1-dim. table defined by matrix"
       extends Modelica.Icons.Function;
       input ExternalCombiTable1D externalCombiTable1D;
     external "C" ModelicaStandardTables_CombiTable1D_close(externalCombiTable1D)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end destructor;
 
   end ExternalCombiTable1D;
@@ -209,14 +209,14 @@
             extrapolation,
             verboseRead,
             delimiter,
-            nHeaderLines) annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+            nHeaderLines) annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end constructor;
 
     function destructor "Terminate 2-dim. table defined by matrix"
       extends Modelica.Icons.Function;
       input ExternalCombiTable2D externalCombiTable2D;
     external "C" ModelicaStandardTables_CombiTable2D_close(externalCombiTable2D)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaStandardTables.h\"", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library={"ModelicaStandardTables", "ModelicaIO", "ModelicaMatIO", "zlib"});
     end destructor;
 
   end ExternalCombiTable2D;
diff -ruN a/Magnetic/QuasiStatic/FundamentalWave/Examples/BasicMachines/InductionMachines/IMC_Transformer.mo b/Magnetic/QuasiStatic/FundamentalWave/Examples/BasicMachines/InductionMachines/IMC_Transformer.mo
--- a/Magnetic/QuasiStatic/FundamentalWave/Examples/BasicMachines/InductionMachines/IMC_Transformer.mo
+++ b/Magnetic/QuasiStatic/FundamentalWave/Examples/BasicMachines/InductionMachines/IMC_Transformer.mo
@@ -268,5 +268,6 @@
                                             Text(
                   extent={{80,-92},{160,-100}},
                   textStyle={TextStyle.Bold},
-          textString="%m phase transient")}));
+          textString="%m phase transient")}),
+  __OpenModelica_simulationFlags(nls = "newton"));
 end IMC_Transformer;
diff -ruN a/Magnetic/QuasiStatic/FundamentalWave/Examples/BasicMachines/InductionMachines/IMC_YD.mo b/Magnetic/QuasiStatic/FundamentalWave/Examples/BasicMachines/InductionMachines/IMC_YD.mo
--- a/Magnetic/QuasiStatic/FundamentalWave/Examples/BasicMachines/InductionMachines/IMC_YD.mo
+++ b/Magnetic/QuasiStatic/FundamentalWave/Examples/BasicMachines/InductionMachines/IMC_YD.mo
@@ -198,5 +198,6 @@
           textString="%m phase quasi-static"), Text(
                   extent={{-60,-80},{20,-88}},
                   textStyle={TextStyle.Bold},
-                  textString="%m phase transient")}));
+                  textString="%m phase transient")}),
+  __OpenModelica_simulationFlags(nls = "newton"));
 end IMC_YD;
diff -ruN a/Math/FastFourierTransform.mo b/Math/FastFourierTransform.mo
--- a/Math/FastFourierTransform.mo
+++ b/Math/FastFourierTransform.mo
@@ -642,7 +642,7 @@
     protected
       Real work[3*size(u,1) + 2*(div(size(u,1),2)+1)];
       external "C" info = ModelicaFFT_kiss_fftr(u, size(u,1), work, size(work,1), amplitudes, phases)
-                   annotation(IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaFFT.h\"", Library="ModelicaExternalC");
+                   annotation(IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library="ModelicaExternalC");
       annotation (Documentation(revisions="<html>
 <table border=\"1\" cellspacing=\"0\" cellpadding=\"2\">
 <tr><th>Date</th> <th align=\"left\">Description</th></tr>
diff -ruN a/Math/Random.mo b/Math/Random.mo
--- a/Math/Random.mo
+++ b/Math/Random.mo
@@ -222,7 +222,7 @@
         output Integer stateOut[nState]
           "The new internal states of the random number generator";
         external "C" ModelicaRandom_xorshift64star(stateIn, stateOut, result)
-          annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaRandom.h\"", Library="ModelicaExternalC");
+          annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library="ModelicaExternalC");
         annotation(Documentation(info="<html>
 <h4>Syntax</h4>
 <blockquote><pre>
@@ -395,7 +395,7 @@
         output Integer stateOut[nState]
           "The new internal states of the random number generator";
         external "C" ModelicaRandom_xorshift128plus(stateIn, stateOut, result)
-          annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaRandom.h\"", Library="ModelicaExternalC");
+          annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library="ModelicaExternalC");
         annotation (Documentation(info="<html>
 <h4>Syntax</h4>
 <blockquote><pre>
@@ -572,7 +572,7 @@
         output Integer stateOut[nState]
           "The new internal states of the random number generator";
         external "C" ModelicaRandom_xorshift1024star(stateIn, stateOut, result)
-          annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaRandom.h\"", Library="ModelicaExternalC");
+          annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library="ModelicaExternalC");
         annotation (Documentation(info="<html>
 <h4>Syntax</h4>
 <blockquote><pre>
@@ -935,7 +935,7 @@
       extends Modelica.Icons.Function;
       output Integer seed "Automatically generated seed";
 
-      external "C" seed = ModelicaRandom_automaticGlobalSeed(0.0) annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaRandom.h\"", Library="ModelicaExternalC");
+      external "C" seed = ModelicaRandom_automaticGlobalSeed(0.0) annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library="ModelicaExternalC");
      annotation (Documentation(info="<html>
 <h4>Syntax</h4>
 <blockquote><pre>
@@ -1073,7 +1073,7 @@
         input Integer[33] rngState "The initial state";
         input Integer id;
         external "C" ModelicaRandom_setInternalState_xorshift1024star(rngState, size(rngState,1), id)
-          annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaRandom.h\"", Library="ModelicaExternalC");
+          annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library="ModelicaExternalC");
       end setInternalState;
 
     algorithm
@@ -1153,7 +1153,7 @@
       output Real y
         "A random number with a uniform distribution on the interval (0,1]";
       external "C" y = ModelicaRandom_impureRandom_xorshift1024star(id)
-        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Include="#include \"ModelicaRandom.h\"", Library="ModelicaExternalC");
+        annotation (IncludeDirectory="modelica://Modelica/Resources/C-Sources", Library="ModelicaExternalC");
       annotation(Documentation(info="<html>
 <h4>Syntax</h4>
 <blockquote><pre>
